apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -302,6 +305,7 @@ export const ModerationNotification = (props) => {
           </div>}
 
         <p>{commentText}</p>
+        <AIFeedback AIReport={translated.dummyText} />
```

Comment:
this will actually come from the serializer but until we have it I just added a string

Reply:
A semantic note: Hannes started by namimg the module AiReport. It is helpful to remain consistent because when we are looking up AiReport we find the instances of this name. So either we all change it to AIReport/AIFeedback, or you change it here to AiReport/AiFeedback @goapunk @hklarner